src/php/ext/grpc/channel_credentials.c

```diff
@@ -116,8 +116,13 @@ PHP_METHOD(ChannelCredentials, setDefaultRootsPem) {
                          "setDefaultRootsPem expects 1 string", 1 TSRMLS_CC);
     return;
   }
-  default_pem_root_certs = gpr_malloc((pem_roots_length + 1) * sizeof(char));
-  memcpy(default_pem_root_certs, pem_roots, pem_roots_length + 1);
+  char *default_certs = php_grpc_get_default_pem_root_certs(TSRMLS_C);
+  if (default_certs == NULL) {
```

Comment:
I don't understand this change.

AFAICT, you've added a "PHP global" for pem root certs, will set that the first time this function gets called, and subsequent calls to this function will do nothing.

Is that the intended behavior?  Also, if that's whats needed, wouldn't it be simpler to  use ```default_pem_root_certs``` to determine if it has been set?